### DISCUSSION TITLE
[功能建议]港澳台版权番剧播放时导入大陆版权番剧（若有）弹幕

### DISCUSSION THREAD

**Discussion by Q16KBreak** [the1812/Bilibili-Evolved#2327] [功能建议 / Ideas]
如题

**Comment by Geeyun-JY3** [comment]
港澳台相关建议还是自力更生

**Comment by CKylinMC** [comment]
你可以想办法下载下来，用dandanplay放，那个甚至能抓全网弹幕...

**Reply by Q16KBreak** [reply]
dandanplay的弹幕系统感觉没有b站的舒服 也可能是先入为主了吧

**Reply by CKylinMC** [reply]
> dandanplay的弹幕系统感觉没有b站的舒服 也可能是先入为主了吧

可能是看习惯了吧，我其实也看b站字幕习惯一点。那个字幕样式我记得是可调的，试试调节一下没准会好一些，问题不大...
